<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectSaveLoacation</class>
 <widget class="QWidget" name="SelectSaveLoacation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>where do you want to save</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="save1_pb">
      <property name="text">
       <string>save 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save2_pb">
      <property name="text">
       <string>save 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save3_pb">
      <property name="text">
       <string>save 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save4_pb">
      <property name="text">
       <string>save 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save5_pb">
      <property name="text">
       <string>save 5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clear_pb">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>clear all saves</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="clear1">
      <property name="text">
       <string>clear 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear2">
      <property name="text">
       <string>clear 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear3">
      <property name="text">
       <string>clear 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear4">
      <property name="text">
       <string>clear 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear5">
      <property name="text">
       <string>clear 5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="save1_lbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>115</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>available</string>
   </property>
  </widget>
  <widget class="QLabel" name="save2_lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>115</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>available</string>
   </property>
  </widget>
  <widget class="QLabel" name="save3_lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>115</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>available</string>
   </property>
  </widget>
  <widget class="QLabel" name="save4_lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>115</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>available</string>
   </property>
  </widget>
  <widget class="QLabel" name="save5_lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>115</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>available</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
